reference/7.0/Microsoft.PowerShell.Core/About/about_Telemetry.md

```diff
@@ -14,35 +14,42 @@ Describes the telemetry collected in PowerShell and how to opt-out.
 
 ## Long description
 
-PowerShell sends basic telemetry data to Microsoft. This data allows us to
-better understand the environments where PowerShell is used and enables us to
-prioritize new features and fixes. The telemetry is includes anonymized
-information about the host running PowerShell, and information about how
-PowerShell is used.
+PowerShell sends basic telemetry data to Microsoft using Application Insights.
+This data allows us to better understand the environments where PowerShell is
+used and enables us to prioritize new features and fixes. The telemetry is
+includes anonymized information about the host running PowerShell, and
+information about how PowerShell is used.
 
-The host-based telemetry data includes:
+PowerShell send the following information at startup:
 
-- OS name
-- OS version
-- PowerShell version
-- The PowerShell distribution channel
+- The name and version of the operating system
+- The version of PowerShell
 - The version of the Application Insights SDK used by PowerShell
 - The geographic location of the host, based on the IP address
+- A randomly generated GUID representing the user running the instance
+- A randomly generated GUID representing the session instance
+
+Startup telemetry data is only collected when starting the `pwsh` executable.
+This information isn't sent if PowerShell is embedded in some other host
+application.
 
 > [!NOTE]
-> Application Insights uses the hosts IP address to determine the geographic
-> location. The IP address is never included in the telemetry data or stored
+> Application Insights uses the hosts IP Address to determine the geographic
+> location. The IP Address is never included in the telemetry data or stored
```

Comment:
you could add city, state, or region

Reply:
You mean instead of "location"? I am not sure what you are trying to say.

Reply:
I added links to the AppInsights docs so we don't have to say that here.